src/webapi/Evaluations/Data/IEvaluationRepository.cs

```diff
@@ -12,6 +12,7 @@ namespace eppeta.webapi.Evaluations.Data;
 public interface IEvaluationRepository
 {
     Task<List<Evaluation>> GetAllEvaluations();
+    Task<List<PerformanceEvaluation>> GetAllPerformanceEvaluationsById(int performanceEvaluationId);
```

Comment:
Rename to GetPerformanceEvaluationById

Reply:
@stevenarnold-ed-fi Technically the method returns the list of matching PerformanceEvaluations by Id. However, since Id is the PK it will always return a single record. To make this change coherent the signature should be changed to:

`Task<PerformanceEvaluation> GetAllPerformanceEvaluationById(int performanceEvaluationId);`

Do you agree?

Reply:
I missed that it was returning a list of PerformanceEvaluations, this should be changed as well. But the All should be removed from the name so:
Task<PerformanceEvaluation> GetPerformanceEvaluationById(int performanceEvaluationId);
would work